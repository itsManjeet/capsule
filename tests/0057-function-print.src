let foo = fun () { };

println (foo);