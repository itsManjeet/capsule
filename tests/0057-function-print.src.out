0000 CONST_NULL | 0001 RET | 
